comp/metadata/packagesigning/utils/aptutils.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package utils
```

Comment:
Looks like the utils files have the suffix utils, but they are groupd in the utils folder. We might be able to remove the utils suffix from the file name: `comp/metadata/packagesigning/utils/apt.go`

Reply:
I fully agree!